packages/deployer/test/sample-project/test/OwnerModule.test.js

```diff
@@ -0,0 +1,108 @@
+const hre = require('hardhat');
+const assert = require('assert');
+const { ethers } = hre;
+const { getProxyAddress } = require('../../../utils/deployments');
+const { assertRevert } = require('@synthetixio/core-js/utils/assertions');
+const { bootstrap } = require('./helpers/initializer');
+
+describe('OwnerModule', () => {
+  bootstrap();
+
+  let OwnerModule;
+
+  let owner, user;
+
+  before('identify signers', async () => {
+    [owner, user] = await ethers.getSigners();
+  });
+
+  before('identify modules', async () => {
+    const proxyAddress = getProxyAddress();
+
+    OwnerModule = await ethers.getContractAt('OwnerModule', proxyAddress);
+  });
+
+  describe('before an owner is set or nominated', () => {
+    it('shows that no owner is set', async () => {
+      assert.equal(await OwnerModule.getOwner(), '0x0000000000000000000000000000000000000000');
```

Comment:
What do you think of creating an `ZERO_ADDRESS` constant? I've been seeing this throughout all the code and it could be defined on `core-js`, to avoid having to copy paste it.

Reply:
Yeah, this is a constant point of debate. Some people prefer that kind of constants. I prefer to be explicit. Once you become familiar with addresses, its easier and more explicit to just enter 0x followed by 40 zeroes to represent the zero address. Let's leave it out of this PR, and if you insist, create an issue in the sprint?